src/ProjectTemplates/test/ProjectTemplates.Tests.csproj

```diff
@@ -4,9 +4,6 @@
     <DefineConstants>$(DefineConstants);XPLAT</DefineConstants>
     <!-- WebDriver is not strong-named, so this test project cannot be strong named either. -->
     <SignAssembly>false</SignAssembly>
-
-    <!-- Workaround until https://github.com/aspnet/AspNetCore/issues/4321 is resolved. -->
-    <SkipTests Condition="'$(RunTemplateTests)' != 'true'">true</SkipTests>
```

Comment:
@ryanbrandenburg I added to this PR. It passes on all but one test for me. If I can't figure it out, I'm going to revert this so tests are still skipped in templating.

```
 Failed   Templates.Test.RazorComponentsTemplateTest.RazorComponentsTemplateWorks

error : NETSDK1061: The project was restored using Microsoft.AspNetCore.App version 3.0.0-preview-18572-0018, but with current settings, version 3.0.0-alpha1-
  10663 would be used instead. To resolve this issue, make sure the same settings are used for restore and for subsequent operations such as build or publish. Typically thi
  s issue can occur if the RuntimeIdentifier property is set during build or publish but not during restore. For more information, see https://aka.ms/dotnet-runtime-patch-s
  election.

```

Reply:
Looks like there was some workaround code in the test that was unnecessary. I've removed it and the test passed locally.